Assets/KoboldKare/Scripts/PrecisionGrabber.cs

```diff
@@ -560,6 +560,11 @@ public void TryGrab() {
     [PunRPC]
     private void FreezeRPC(int grabViewID, int colliderNum, Vector3 localColliderPosition, Vector3 localHitNormal, Vector3 worldAnchor, Quaternion rotation, bool affRotation) {
         PhotonView grabView = PhotonNetwork.GetPhotonView(grabViewID);
+        if(grabView.GetComponentInChildren<Freezable>()!=null)
+            {
+                grabView.RPC("FreezeRPC", RpcTarget.All); 

```

Comment:
This should be `grabView.RPC(nameof(Freezable.FreezeRPC), RpcTarget.All);` so the compiler can handle the string.